addons/l10n_ch/__init__.py

```diff
@@ -5,6 +5,26 @@
 
 from odoo import api, SUPERUSER_ID
 
-def load_translations(cr, registry):
-    env = api.Environment(cr, SUPERUSER_ID, {})
+
+def load_translations(env):
     env.ref('l10n_ch.l10nch_chart_template').process_coa_translations()
+
+
+def init_settings(env):
+    '''If the company is localized in Switzerland, activate the cash rounding by default.
+    '''
+    company_id = env.user.company_id
```

Comment:
this function should care about all companies located in Switzerland, not only the company of the user